frontend/app/routes/_gcweb-app.letters._index.tsx

```diff
@@ -28,15 +27,14 @@ export async function loader({ request }: LoaderFunctionArgs) {
    * @sort This accesses the URL's search parameter and retrieves the value associated with the 'sort' parameter, allows the client to specify how the data should be sorted via the URL
    */
   const url = new URL(request.url);
-  const sortOrder = orderEnumSchema.catch('desc').parse(url.searchParams.get('sort'));
+  const sortOrder = orderEnumSchema.parse(url.searchParams.get('sort') || 'desc');
```

Comment:
https://zod.dev/?id=catch
`.catch('desc')` will set the parse default to `desc` if there is an error while parsing.